Firestore/core/include/firebase/firestore/timestamp.h

```diff
@@ -145,10 +145,16 @@ class Timestamp {
    * Returns a string representation of this `Timestamp` for logging/debugging
    * purposes.
    *
-   * Note: the exact string representation is unspecified and subject to change;
-   * don't rely on the format of the string.
+   * @note: the exact string representation is unspecified and subject to
+   * change; don't rely on the format of the string.
    */
   std::string ToString() const;
+
+  /**
+   * Outputs the string representation of this `Timestamp` to the given stream.
+   *
+   * @see `ToString()` for comments on the representation format.
```

Comment:
Optional: Do you think end users are more likely to look for `operator<<` for getting a string representation? If so, maybe we should make the reference go the other way? I'm torn though because that's the reverse to the order in the file. Maybe reorder as well? This seems like a lot of effort for a nit, so please feel free to ignore.

Reply:
I'm not sure; my presumption was that `ToString` is a better place because it's more generic than `operator<<`, but I don't feel strongly about it at all.

Reply:
It's fine to leave as-is. If the comment resonated with you as something that might be a reasonable way to think about this it might have been worth pursuing. Since it doesn't, don't.